packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx

```diff
@@ -18,6 +18,7 @@ import type {
 import { getOutputState } from 'src/utils/wallet/reviewTransactionUtils';
 import { TransactionReviewTotalOutput } from './TransactionReviewTotalOutput';
 import { ReviewOutput } from 'src/types/wallet/transaction';
+import { log } from 'console';
```

Comment:
not needed - also the reason why CI fails

Reply:
:heavy_check_mark: